File: src/test/java/cn/enaium/cf4m/test/module/module/ModuleTypeTest.java
package cn.enaium.cf4m.test.module.module;

import cn.enaium.cf4m.CF4M;
import cn.enaium.cf4m.annotation.module.Enable;
import cn.enaium.cf4m.annotation.module.Module;

/**
 * @author Enaium
 */
@Module(value = "ModuleTypeTest", key = 5)
public class ModuleTypeTest {
    @Enable
    public void enable() {
        for (String s : CF4M.MODULE.getAllType()) {
            System.out.println(s);
        }
    }
}

File: src/test/java/cn/enaium/cf4m/test/module/module/ModuleTypeTest.kt
package cn.enaium.cf4m.test.module.module

import cn.enaium.cf4m.CF4M
import cn.enaium.cf4m.annotation.module.Enable
import cn.enaium.cf4m.annotation.module.Module

/**
 * Project: cf4m
 *
 * @author Enaium
 */
@Module("ModuleTypeTest", key = 5)
class ModuleTypeTest {
    @Enable
    fun enable() {
        for (s in CF4M.INSTANCE.module.allType) {
            println(s)
        }
    }
}